db/auth/docker-compose.migrations.yml

```diff
@@ -10,7 +10,7 @@ services:
         "up"
         ]
     volumes:
-      - ./db/migrations:/migrations
+      - ./migrations:/migrations
```

Comment:
It would be better if you do it from apps

Reply:
why? this docker-compose only for creation new migrations by developer. 
this method is indicated in doc
[https://github.com/golang-migrate/migrate](url)

In app we apply migrations it the other way
<img width="602" alt="Снимок экрана 2023-07-30 в 11 35 40" src="https://github.com/kripsy/gophermart/assets/3986620/b46fa35f-aa91-4de2-81e3-222d26eecdd6">

Reply:
> Because I need to run several docker compose files.

Есть компос для разработки, есть для развертывания.
Над компосом для разработки мы не заморачивались. Более того, большинство, я уверен, просто вызывали docker команду. Переделывать на этапе сдачи не очень хотелось бы.

На счет embed filesystem - да, согласен, можно аттачить файл через директиву go. Но можно и так.